### ISSUE TITLE
Setup

### ISSUE THREAD

**Issue by vongohren** [reinbach/strider-stash#1]
It was not as easy as you say it was, maybe things have changed?
But I would like to discuss with you how to set things up, since the README was not complete.

**Comment by reinbach** [comment]
Hi @Snorlock what exactly is the issue you are having? What steps did you take and what is your result?

**Comment by vongohren** [comment]
Sorry for late reply but this was but on hold. Im struggeling to recieve the access toke form stash:

500 InternalOAuthError: failed to obtain request token

I have created the certs to talk with stash and have full controll over them.
But the application link process might have changed. First of all your link does not lead to anything, just the overall help.
And when I find the application link setup which I think you want to point to: https://confluence.atlassian.com/display/APPLINKS/Application+Links+User+Guide
I still struggle to get it to work.
There is alot of information that is not completly obvious to me that needs to be filled in.

Feks, do I need port to the URL. It want some specific OAuth urls in the setup. And more

**Comment by reinbach** [comment]
Hi @Snorlock it looks like their docs have been changed, since I added that link. It has been a while since I've used Stash.
I pulled the following instructions from another project. See if they help at all, if they do, I'll update these docs as well.;

Setup Application Link

In Administration Settings section, configure an Application Link.

Step 1.

Application Name: go-stash
Application Type: Generic Application
Create Incoming Link: checked

Step 2.

Consumer Key: <consumer_key>
Consumer Name: <consumer_name>
Public Key: <public_key>
